exercises/concept/log-line-parser/.docs/introduction.md

```diff
@@ -54,8 +54,8 @@ str[1, 3]         #=> "ome"
 str.slice(1, 3)   #=> "ome"
 ```
 
-The semantic values of strings can also be maniplulated.
-It's worth exploring the Ruby docs to learn about all the methods, but here are a few commonly used examples:
+The semantic values of strings can also be manipulated.
+It's worth exploring the [Ruby docs][docs-string] to learn about all the methods, but here are a few commonly used examples:
```

Comment:
I could have used a link here. It's not entirely clear at this point how to find the Ruby docs. There's a link in the hints, I feel like this is too fundamental to the concept to "hide" in the hints 🤔

Reply:
Agreed.